docs/curriculum/detailed-schedule.md

```diff
@@ -58,7 +63,7 @@ take 25 minutes. Join us for the walk if you want, or otherwise meet us in the d
 | 3:30  | 4:00  | Intermediate workflows (lecture)                                      | Lauren Michael           |
 | 4:00  | 5:00  | Running workflows locally with HTCondor and DAGMan, Part 2 (hands-on) | "                        |
 | 5:00  | —     | On your own                                                           |                          |
-| 7:00  | 9:00  | Evening work session                                                  | Bala, Derek, Lauren, Tim |
+| 7:00  | 9:00  | Evening work session (at Union South, search the “Today In The Union” (TITU) display for the exact room)  | Bala, Derek, Lauren, Tim |
```

Comment:
Would it be better to put the long comment after the table, so as not to destroy the table formatting?